vendorweapons.php

```diff
@@ -8,35 +9,75 @@
 ?>
-<html>
- <head>
-  <title>Welcome to /r/DTG/</title>
-  <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
-  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.15/css/jquery.dataTables.css">  
-  
-  <script src="https://code.jquery.com/jquery-2.2.4.min.js" integrity="sha256-BbhdlvQf/xTY9gja0Dq3HiwQF8LaCRTXxZKRutelT44=" crossorigin="anonymous"></script>
-  <script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.10.15/js/jquery.dataTables.js"></script>
-  <style>    
-         * {
-                font-size: 13px;
-                line-height: 1.4;
-        }
-        
-  </style>
- </head>
- <body>
-     <div class="container">
-     <h1>Vendor Weapons</h1>
-     <?php
+    <html>
+
+    <head>
+        <title>Welcome to /r/DTG/</title>
+        <!-- CSS -->
+        <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+        <link rel="stylesheet" type="text/css" href="./static/css/main.css">
+        <link rel="stylesheet" type="text/css" href="./static/css/nav.css?ver=3">
+        <link rel="stylesheet" type="text/css" href="./static/css/tables.css?ver=5">
+        <link rel="stylesheet" type="text/css" href="./static/css/jquery.dataTables.css">
+        <!-- JS -->
+        <script src="./static/js/jquery-3.2.1.min.js"></script>
+        <script src="./static/js/bootstrap.min.js"></script>
+        <script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.10.15/js/jquery.dataTables.js"></script>
+    </head>
+
+    <body>
+        <div class="masthead-weapons"> </div>
+        <div id="nav-wrap">
+            <div id="main-nav" class="navbar">
+                <div class="navbar-inner">
+                    <div class="container">
+                        <div class="navbar-header">
+                            <div class="navbar-brand navbar-brand-centered"></div>
+                        </div>
+                    </div>
+                </div>
+                <!-- navbar -->
+            </div>
+        </div>
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-md-2 sidebar1">
+                    <div class="container sidebar1" id="sidebar">
+                        <div class="logo"> <img src="./static/img/Raid_Emblem.png" class="rounded" alt="Logo" style="max-width:175px; border:0; padding-left: 40px;"> </div>
+                        <br>
+                        <br>
+                        <br>
+                        <br>
+                        <div class="left-navigation">
+                            <ul class="list">
+                                <h5><strong>LINKS</strong></h5>
+                                <li><a href="https://goo.gl/xgpzbX">Discuss this on Reddit</a></li>
+                                <li><a href="vendorweapons.php">Vendor Weapons</a></li>
```

Comment:
Change this to reference the Vendor Armor since we're already on the Weapons page.